cmd/bosun/sched/sched.go

```diff
@@ -537,6 +537,10 @@ const (
 	dbMetadata      = "metadata"
 )
 
+func init() {
+	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes, "Total size of the bosun state file.")
```

Comment:
"The total size" , otherwise I don't think it is a complete sentence? I gotz no grammar skills, I'm-a-guessin based on it not sounding right.